radixdlt/src/main/java/com/radixdlt/consensus/VertexStore.java

```diff
@@ -86,7 +86,9 @@ public void insertVertex(Vertex vertex) throws VertexInsertionException {
 			try {
 				this.engine.store(vertex.getAtom());
 			} catch (RadixEngineException e) {
-				throw new VertexInsertionException("Failed to execute", e);
+				// TODO: Don't check for state computer errors for now
```

Comment:
What's the though process behind this?

It seems pretty error prone to me, as it looks like ValidatingEventCoordinator will no longer remove atoms that fail to store from the memory pool.

Reply:
they will get removed once the vertex is committed

Reply:
Would be good to elaborate your reasoning in this comment here as it looks broken unless you know that